addons/mail/static/src/components/rtc_controller/rtc_controller.xml

```diff
@@ -55,6 +55,7 @@
                         <t t-if="messaging.rtc.sendDisplay"><t t-set="displayTitle">Stop screen sharing</t></t>
                         <t t-else=""><t t-set="displayTitle">Share screen</t></t>
                         <button class="o_RtcController_button o_RtcController_videoButton"
+                            t-if="!env.device.isMobileDevice"
```

Comment:
```suggestion
                            t-if="!messaging.device.supportsScreenSharing"
```
and then add this field to the [`Device` model](https://github.com/odoo/odoo/blob/master/addons/mail/static/src/models/device.js) like this:

in `recordMethods`:
```JS
_computeSupportsScreenSharing() {
    return Boolean(navigator.mediaDevices && navigator.mediaDevices.getDisplayMedia);
},
```
in `fields`:
```JS
supportsScreenSharing: attr({
    compute: '_computeSupportsScreenSharing',
}),
```

like this check for rtc: https://github.com/odoo/odoo/blob/master/addons/mail/static/src/models/device.js#L51

Reply:
Nope... getDisplayMedia exits as you can see in the documentation. But doesn't work as expected....

Reply:
> Nope... getDisplayMedia exits as you can see in the documentation. But doesn't work as expected....

@adr-odoo 

I see that it's just for firefox mobile, is it acceptable to let them have the current message (which is correct since it seems that firefox sends this `NotAllowedError` error)? The other mobile browsers do not have the `getDisplayMedia`. 

If not, let's do it like that, but it will require extra attention to unlock the feature as mobile browsers start supporting it.

Reply:
No, users submit tickets because there are no able to add this permission... Anyway, we have to allow this permission in the mobile app.. So, we will unlock this in mail in the same time.